tensorflow_graphics/datasets/features/pose_feature.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2020 The TensorFlow Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# Lint as: python3
+"""3D Pose feature."""
+
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import numpy as np
+import six
+import tensorflow.compat.v2 as tf
+from tensorflow_datasets import features
+from tensorflow_graphics.io import triangle_mesh
+
+
+class Pose(features.FeaturesDict):
+  """`FeatureConnector` for 3d pose representations.
+
+  During `_generate_examples`, the feature connector accepts as input any of:
+
+    * `dict:` A dictionary containing the rotation and translation of the object (see
+       output format below).
+
+  Output:
+    A dictionary containing:
+
+    * 'rotation': A `float32` tensor with shape `[3, 3]` denoting the 3D rotation matrix.
+    * 'translation': A `float32` tensor with shape `[3,]` denoting the translation vector.
+
+  """
+  def __init__(self):
+    super(Pose, self).__init__({
+        'R': features.Tensor(shape=(3, 3), dtype=tf.float32),
+        't': features.Tensor(shape=(3,), dtype=tf.float32),
+    })
+
+  def encode_example(self, pose_dict):
+    """Convert the given pose into a dict convertible to tf example."""
+    assert list(pose_dict.keys()) >= ['R', 't'], "Missing keys in provided dictionary! Expecting 'R' and 't'."
```

Comment:
raise ValueError instead of assert. see  for eg [here](https://github.com/tensorflow/graphics/blob/830515c44354e2d013866c08360fca875a443359/tensorflow_graphics/io/triangle_mesh.py#L68)